docs/setup/upgrade/logging-configuration-migration.asciidoc

```diff
@@ -1,13 +1,32 @@
 [[logging-configuration-migration]]
 == Logging configuration migration
 
+[float]
+=== Prepare for logging in 8.0
+
+The entire logging system changed to offer a more consistent logging behavior with {es}. The current system differs from the more simplistic, deprecated system and allows you the freedom to customize what, how and where {kib} outputs log messages. While for logging based use cases, we recommend the <<logging-service, logging service guide>>, there are some important considerations:
+
+[float]
+==== Mixing deprecated and current configurations
+{kib} supports mixed usage of the deprecated and current logging configuration styles, however, we recommend changing them all at once to reduce the risk of the systems interfering.
+
+[float]
+==== Rotating log files
+If you use log rotation, be careful when mixing deprecated and current configuration styles as it could lead to unexpected behavior. For example, one system could apply the log file rotation while the other still writes to the original file. Rather switch over to the current system completely.
```

Comment:
grammar nit:
```suggestion
If you use log rotation, be careful when mixing deprecated and current configuration styles as it could lead to unexpected behavior. For example, one system could apply the log file rotation while the other still writes to the original file. Rather, switch over to the current system completely.
```